pkg/state/common_test.go

```diff
@@ -169,6 +193,25 @@ func assertCNIBinDirForDS(u *unstructured.Unstructured) {
 	}
 }
 
+func assertNetworkAttachmentDefinition(c client.Client, expectedNadConfig *nadConfig,
```

Comment:
Q: Should we use here `GinkgoHelper()` or `ExpectWithOffset`?

Reply:
This would be helpful for sure. We are never using `GinkgoHelper` or `ExpectWithOffset` in our tests.
Should we create a separate PR to align our tests?

Reply:
yes, I think we can update all helper functions in a separate PR